docs/resources/private_crossconnect.md

```diff
@@ -13,10 +13,36 @@ Manages a **Private Cross Connect** on IonosCloud.
 
 ## Example Usage
 
+To connect two datacenters, we need 2 lans defined one in each datacenter. After, we reference the pcc through which we want the connection to be made established.
```

Comment:
@cristiGuranIonos I suggest adding a come here, something like: `we need 2 LANs defined, one in each datacenter. Also, `made` or `established` at the end of the sentence, not both.